support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc

```diff
@@ -7,6 +7,9 @@ toc::[]
 
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 You may choose to opt out of reporting health and usage data for your cluster.
+
+Alternatively, you can obfuscate the IPv4 addresses and cluster base domains Insights Operator sends to link:https://cloud.redhat.com[cloud.redhat.com] (this feature is disabled by default). See xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#insights-operator-enable-ofuscation_opting-out-remote-health-reporting[Enabling Insights Operator data obfuscation] for details.
```

Comment:
[cloud.redhat.com]. This feature is disabled by default.

Reply:
Fixed